packages/admin-ui/src/filters/filter-popover.tsx

```diff
@@ -12,11 +12,17 @@ import { FilterStatus } from './filter-status'
 
 export const FilterPopover = createComponent<typeof Menu, FilterPopoverOptions>(
   (props) => {
-    const { children, state, ...restProps } = props
+    const { children, state, onRetry = () => {}, ...restProps } = props
 
     const [isScrollableLayout, setIsScrollableLayout] = useState(false)
     const contextState = { isScrollableLayout, setIsScrollableLayout, state }
 
+    const shouldDisplayChildren = !(
+      state.status === 'error' ||
+      state.status === 'loading' ||
+      state.status === 'empty'
+    )
```

Comment:
Is it the same result?
```suggestion
    const shouldDisplayChildren = state.status === 'ready'
```

Reply:
it seems to have others status, but I am not entirely sure 🤔

Reply:
No, all the search status should still display children